github.go

```diff
@@ -202,6 +212,39 @@ func createGitHubRepoUrlForPath(repo GitHubRepo, path string) string {
 	return fmt.Sprintf("repos/%s/%s/%s", repo.Owner, repo.Name, path)
 }
 
+// Get the Next paginated path from the link url api.github.com/repos/:owner/:repo/:path
+// If there is no next page, return an empty string
+// Links are formatted: "<url>; rel=next, <url>; rel=last"
+func getNextPath(links string) (string, *FetchError) {
+	if len(links) == 0 {
+		return "", nil
+	}
+
+	nextLinkRegex, regexErr := regexp.Compile(`<(.+?)>;\s+rel="next"`)
```

Comment:
In the regex here, it's `rel="next"` (in double quotes), but in the comment on line 217, it's `rel=next` (no double quotes). Which is correct?

Reply:
NIT: Also, this regex can be defined outside of this function once (e.g., at the top of `github.go`), with `MustCompile`, rather than each time this function is called.